modules/spock/src/test/groovy/org/testcontainers/spock/TestLifecycleAwareIT.groovy

```diff
@@ -7,12 +7,12 @@ import spock.util.EmbeddedSpecRunner
 
 class TestLifecycleAwareIT extends Specification {
 
-    @Unroll("When failing test is #fails, afterTest receives '#filesystemFriendlyNames' and throwable starting with message '#errorMessageStartsWith'")
-    def "lifecycle awareness"() {
-        given:
+	@Unroll("When failing test is #fails, afterTest receives '#filesystemFriendlyNames' and throwable starting with message '#errorMessageStartsWith'")
+	def "lifecycle awareness"() {
+		given:
 
-        @Language("groovy")
-        String myTest = """
+		@Language("groovy")
+				String myTest = """
```

Comment:
Those lines should be aligned. But if it is hard to configure, we can live with that.